pkg/spec/parse.go

```diff
@@ -144,6 +144,18 @@ func getLoggingPath(opts []string) string {
 	return ""
 }
 
+func getLoggingTag(opts []string) string {
```

Comment:
Can we combine this and getLoggingPath into one function, so we don't have to double-traverse the parameters? We can also add warnings for unrecognized options

Reply:
done